app/renderer/components/preferences/payment/addFundsDialog/steps/addFundsBatScreen.js

```diff
@@ -18,10 +19,13 @@ class BatWelcomeScreen extends React.Component {
         className={css(styles.batScreen)}
       >
         <AboutPageSectionTitle data-canWrap data-l10n-id='helloBat' />
-        <p data-l10n-id='helloBatText1'
+        <div data-l10n-id='helloBatText1'
```

Comment:
any reason why not paragraph?

Reply:
not really. I just thought I have not seen `p` many times under `app/renderer/components` (we have a lot of `div` and `span` instead of `p`) and standardizing would help us to keep consistency.

If it should be left as `p`, I'll revert them and that's no problem for me :-)

Reply:
ya not sure why but please revert this part, the current way is not semantic